### DISCUSSION TITLE
How crop picture vertically

### DISCUSSION THREAD

**Discussion by s01o** [ImageMagick/ImageMagick#4245] [Help] [answered]
`convert 101.png -crop 96x64 out\00.png`

How can I make this command crop the picture vertically 

1 4 7 10
2 5 8 11
3 6 9 12

sequence like this

**Comment by snibgo** [comment, marked as answer]
What version of IM, on what platform (bash, Windows, etc)?
```
magick in.png -transpose -crop 3x4@ -transpose out-%02d.png
```
This creates output files out-00.png, out-01.png, etc, in the order you want.

**Reply by s01o** [reply]
grateful
